requirements/fabric/strategies.txt

```diff
@@ -1,3 +1,3 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
-deepspeed >=0.8.2, <=0.9.1; platform_system != "Windows"
+deepspeed >=0.8.2, <=0.9.3; platform_system != "Windows"
```

Comment:
I thought that we wont to bunt also the min version...

Reply:
I think that what you are saying is that we should find which is the minimal version we can support today? Well I think that's out of scope for this PR. The procedure would be to force-downgrade and run the test suite with this version, then iteratively increase the version until all tests pass.